libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```
@@ -5,6 +5,13 @@ $checkbox-icon-size: size('m');
 $spacing-to-edge: map-get($checkbox-radio-spacing, 'to-edge');
 $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
 
+@mixin wrapper-top-bottom-padding($target-height) {
+  $padding: calc((#{$target-height} - #{$checkbox-icon-size}) / 2);
```

Do you need to calculate this at runtime? If not, consider replacing with a built time calc:
```suggestion
  $padding: ($target-height - $checkbox-icon-size) / 2;
```

---

Good point!

---

Good point! - Should be fixed by d28acfae53c7e90e4c133f64bb70af305339fa72